server/rest/src/main/java/org/infinispan/rest/server/operations/mediatypes/printers/Escaper.java

```diff
@@ -0,0 +1,44 @@
+package org.infinispan.rest.server.operations.mediatypes.printers;
+
+public class Escaper {
+
+   private Escaper() {
+
+   }
+
+   static String escapeHtml(String html) {
+      return escapeXml(html);
+   }
+
+   static String escapeXml(String xml) {
```

Comment:
Could you please tell me where I can find it? I know there is something like this in Commons but I'm not sure about JRE.